invenio_stats/aggregations.py

```diff
@@ -345,11 +374,18 @@ def agg_iter(self, dt):
                 interval_date = datetime.strptime(
                     doc["timestamp"], "%Y-%m-%dT%H:%M:%S"
                 ).replace(**dict.fromkeys(INTERVAL_ROUNDING[self.interval], 0))
-
+                if aggregation["last_update"]["value"] and previous_bookmark:
+                    # Let's make sure that both objects are of the same type. This is important for the test
+                    previous = float(previous_bookmark.strftime("%s"))
+                    if aggregation["last_update"]["value"] < previous:
+                        continue
                 aggregation_data = {}
                 aggregation_data["timestamp"] = interval_date.isoformat()
                 aggregation_data[self.field] = aggregation["key"]
                 aggregation_data["count"] = aggregation["doc_count"]
+                aggregation_data["updated_timestamp"] = datetime.utcnow().strftime(
+                    "%Y-%m-%dT%H:%M:%S"
```

Comment:
Very minor: in general, for records, we simply use [the ISO format](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/services/records/schema.py#L27), e.g.:
`"created": "2018-12-13T23:58:22.597431+00:00",`
We can use the native [fromisoformat](https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat) to parse the date.